### PR TITLE
Feature/start activity button

### PR DESCRIPTION
**Pull request by jasmyn2244**
### What (if any) features are you implementing?
- When user clicks "start activity" button, we hide the form and display the countdown timer incorporating the user inputs
### What (if anything) did you refactor?
- We refactored some class names and created helper functions and consolidated them into one function that is called when the "start activity" button is clicked
### Were there any issues that arose?
- No
### Is there anything that you need from your teammate?
- Check it out, make sure we didn't miss any spacing or semi colons
### Refactor checklist
- [x] Check variable and parameter names: are they descriptive and unique?
- [x] Is the code DRY? SRP?
- [x] Is it easy for others to read your code? (explanatory comments needed?)
- [x] Is the syntax consistent? (e.g. use of semicolons, blank lines)
- [x] Are commit messages formatted correctly?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,12 +14,12 @@ <h2 class="page-title">INTENTION TIMER</h2>
...
             <h1 class="form-text-style">Select a category:</h1>
-            <button class="study-button form-text-style category-button-style"><img class="category-button-icons" src="assets/study.svg" alt="lightbulb image"><br>Study</button>
-            <button class="meditate-button form-text-style category-button-style"><img class="category-button-icons" src="assets/meditate.svg" alt="flower image"><br>Meditate</button>
-            <button class="exercise-button form-text-style category-button-style"><img class="category-button-icons" src="assets/exercise.svg" alt="shoe image"><br>Exercise</button>
+            <button class="study-button form-text-style category-button-style"><img class="category-button-icons" src="assets/study.svg" alt="lightbulb image">Study</button>
+            <button class="meditate-button form-text-style category-button-style"><img class="category-button-icons" src="assets/meditate.svg" alt="flower image">Meditate</button>
+            <button class="exercise-button form-text-style category-button-style"><img class="category-button-icons" src="assets/exercise.svg" value="Exercise" alt="shoe image">Exercise</button>
```
Comment (ectweitmann) [inline index.html]: Let's delete this attribute since we aren't using it!

File: main.js [hunk 2]
```diff
@@ -29,12 +38,15 @@ exerciseButton.addEventListener('click', function(event) {
...
-startActivityButton.addEventListener('click', displayErrorMessage);
 
-minutesInput.addEventListener('keydown', preventEInput)
+startActivityButton.addEventListener('click', startActivity);
+
+minutesInput.addEventListener('keydown', preventEInput);
 secondsInput.addEventListener('keydown', preventEInput);
 
 
+var category;
```
Comment (Jayquintana) [inline main.js]: remember to move to the top with the global variable section

File: main.js [hunk 3]
```diff
@@ -8,12 +9,20 @@ var meditateButton = document.querySelector('.meditate-button');
 var exerciseButton = document.querySelector('.exercise-button');
 var buttonImages = document.querySelectorAll('.category-button-icons');
 var startActivityButton = document.querySelector('.start-activity-button');
+var startTimerButton = document.querySelector('.start-timer-button')
```
Comment (ectweitmann) [inline main.js]: Let's add a semi-colon to the end of this statement.

File: main.js [hunk 4]
```diff
@@ -56,11 +69,57 @@ function preventEInput(event) {
...
+  var formFilled = 0;
   for (var i = 0; i < inputs.length; i++) {
     if (!inputs[i].value) {
       errorMessages[i].classList.remove('hidden');
     } else if (inputs[i].value) {
       errorMessages[i].classList.add('hidden');
+      formFilled++;
     }
   }
+  console.log(formFilled);
```
Comment (Jayquintana) [inline main.js]: delete this console log

File: main.js [hunk 5]
```diff
@@ -8,12 +9,20 @@ var meditateButton = document.querySelector('.meditate-button');
...
 var accomplishInput = document.querySelector('.accomplish-input');
 var minutesInput = document.querySelector('.minutes-input');
 var secondsInput = document.querySelector('.seconds-input');
 var inputs = document.querySelectorAll('input');
 var errorMessages = document.querySelectorAll('.error-message');
+var newActivityTitle = document.querySelector('.new-activity-title');
+var timerCountdown = document.querySelector('.timer-countdown');
+var newActivitySection = document.querySelector('.new-activity');
+var currentActivitySection = document.querySelector('.current-activity');
+var intentionActivityTitle = document.querySelector('.intention-activity-title')
```
Comment (ectweitmann) [inline main.js]: Let's add a semi-colon to the end of this statement.

File: styles.css [hunk 6]
```diff
@@ -111,11 +111,16 @@ input {
...
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
   background: none;
   border: 2px solid #FFF;
   border-radius: 5px;
   height: 70px;
   width: 181px;
+
```
Comment (Jayquintana) [inline styles.css]: clear this space

File: main.js (hunk 4 above)
Comment (ectweitmann) [inline main.js]: Let's remove this console.log

**Review by Jayquintana — COMMENTED** [review]
Great jon team

File: main.js [hunk 7]
```diff
@@ -56,11 +69,57 @@ function preventEInput(event) {
...
+function showElement(element) {
+  element.classList.remove('hidden');
+}
+
+function changeText(element, newText) {
+  element.innerText = `${newText}`;
+}
+
+function setTimer() {
+  timerCountdown.innerText = `${minutesInput.value}:${secondsInput.value}`
```
Comment (ectweitmann) [inline main.js]: Let's add a semi-colon to the end of this!

**Review by ectweitmann — CHANGES_REQUESTED** [review]

**Pull request merged**